modules/help_editor/templates/form_edit_help_content.tpl

```diff
@@ -35,10 +28,13 @@
             <div class="col-xs-12"></div>
         {/foreach}
         <br>
-        <input class="btn btn-sm btn-primary col-sm-offset-3" name="fire_away" value="Save" type="submit" />
+        <input type="hidden" id="section" value="{$section}">
+        <input type="hidden" id="subsection" value="{$subsection}">
+        <input type="hidden" id="helpID" value="{$helpID}">
+        <input type="hidden" id="parentID" value="{$parentID}">
+        <input class="btn btn-sm btn-primary col-sm-offset-3" id="save-help" name="fire_away" value="Save" type="submit" />
         <input type="button" name="reset" value="Reset" class="btn btn-sm btn-primary" onclick="location.href='{$baseurl}/help_editor/edit_help_content/?section={$section}&subsection={$subsection}'" />
```

Comment:
It would be nice here if the `helpID` GET parameter was preserved; right now when you click `Reset` a blank help editor page is loaded and you have to manually navigate back to the page you wanted to edit.